scopes.md

```diff
@@ -0,0 +1,138 @@
+- Delete aspects
+- Add/remove person to/from aspects
+
+### conversations
+
+This scope gives access to private messaging, i.e. the following requests:
+- Create conversations
+- Send private messages
+
+### email
+
+This scope is defined by the OpenID Connect spec. It allows reading user's own email address using UserInfo route or from ID Token.
+
+### interactions
+
+This scope gives access to the following requests:
+- Create/delete comments
+- Report posts/comments
+- Post/remove likes
+- Subscribe/mute posts
+- Hide posts from streams
+
+### openid
+
+This scope is defined by the OpenID Connect spec. This is a mandatory scope. It gives access to OpenID Connect authentication.
+
+### private
+
+This scope gives access to the following requests:
+- Read private posts
+- Read private posts intercations (likes, comments)
+
+Reading private posts of contacts additionaly requires `contacts:read` scope.
+
+### public
+
+This scope gives access to the following requests:
+- Read public profiles of users
+- Perform search among publicly searchable profiles
+- Read public posts
+- Read reshares
+- Read public posts intercations (likes, comments)
+
+### posts:publish
+
+This scope gives access to the following requests:
+- Create posts
+- Create reshares
+- Post photos
+
+Create private posts additionaly requires `contacts:read` scope.
+
+### posts:modify
+
+This scope gives access to the following requests:
+- Delete posts
+- Delete photos
+
+### profile
+
+This scope is defined by the OpenID Connect spec. It allows reading user claims (user properties). Claims are read using UserInfo route or from ID Token.
+
+User claims defined by the OpenID Connect are somewhat similar to users own profile data. Generally this scope means "Allow reading user's own profile". Besides user claims defined by the standard this scope also gives access to "read own profile" diaspora* API endpoint.
+
+Supported claims by diaspora:
+
+| claim      | meaning                                             |
+| ---------- | --------------------------------------------------- |
+| name       | Full name                                           |
+| nickname   | The left part of diaspora handle before `@` sign    |
+| profile    | User's profile URL address                          |
+| picture    | User avatar URL address                             |
+| gender     | Gender profile field                                |
+| birthdate  | Birth date profile field                            |
+| locale     | UI language selection specified in account settings |
+| updated_at | Date of last profile edit                           |
+
+### profile:modify
+
+This scope gives access to updating users own profile
+
+### tags
+
+This scope gives access to read/create/delete tag followings.
+
+### streams
+
+This scope gives access streams.
+
+There are 3 types of streams available. Accessing each stream requires two scopes: `streams` scope and a scope specific for the stream type.
```

Comment:
You're right, I'm not sure why I specified it this way. You also didn't mention the [main stream](https://github.com/diaspora/api-documentation/blob/master/_routes/streams.md#get-the-main-stream). This will be fixed :)

Reply:
Yep I missed one too haha